src/main/kotlin/com/aliucord/gradle/plugins/AliucordInjectorGradle.kt

```diff
@@ -0,0 +1,40 @@
+package com.aliucord.gradle.plugins
+
+import com.aliucord.gradle.Constants
+import com.aliucord.gradle.registerDiscordConfiguration
+import com.aliucord.gradle.task.DeployWithAdbTask
+import org.gradle.api.Project
+import org.gradle.api.tasks.Copy
+
+/**
+ * The Gradle plugin used to configure Aliucord's Injector subproject.
+ * ID: `com.aliucord.injector`
+ */
+@Suppress("unused")
+public abstract class AliucordInjectorGradle : AliucordBaseGradle() {
+    override fun apply(project: Project) {
+        registerTasks(project)
+        registerDiscordConfiguration(project)
+    }
+
+    protected fun registerTasks(project: Project) {
+        registerDecompileTask(project)
+
+        // Compilation
+        val compileDexTask = registerCompileDexTask(project)
+
+        // Bundling
+        val makeTask = project.tasks.register("make", Copy::class.java) {
+            group = Constants.TASK_GROUP
+            from(compileDexTask.map { it.outputs.files.singleFile })
+            into(project.layout.buildDirectory)
+            rename { return@rename "Injector.dex" }
```

Comment:
Maybe this "Injector.dex" should be a constant

Reply:
Making it a constant wouldn't help here, this file is only used by the Aliucord repo build workflow to copy it over to the builds branch.